website/docs/r/policy_distributed_flood_protection_profile.html.markdown

```diff
@@ -66,6 +66,8 @@ The following arguments are supported:
 * `description` - (Optional) Description of the resource.
 * `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
 * `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the policy resource.
+* `context` - (Optional) The context which the object belongs to
+  * `project_id` - (Required) The ID of the project which the object belongs to
```

Comment:
extra space ?

Reply:
Hi @alagoutte, what space are you referring to?

Reply:
We have files which have double spacing, e.g:
`website/docs/r/policy_mac_discovery_profile.html.markdown`

And other which have single spacing, e.g:
`website/docs/r/policy_ip_pool_static_subnet.html.markdown`

It might be worthwhile to standardize this although I wouldn't delay this PR for this.